### ISSUE TITLE
Image

### ISSUE THREAD

**Issue by HemingwayLee** [HemingwayLee/test#1]
![auth](https://user-images.githubusercontent.com/8428372/46090268-5279f600-c1eb-11e8-99ef-1494b9f77653.png)
![body](https://user-images.githubusercontent.com/8428372/46090269-5279f600-c1eb-11e8-95a9-4f23698a242b.png)

**Comment by HemingwayLee** [comment]
![from disk cache](https://user-images.githubusercontent.com/8428372/46242703-c6441a80-c406-11e8-99fa-0f7a231084c0.png)

**Comment by HemingwayLee** [comment]
![螢幕快照 2019-06-15 下午4 09 57](https://user-images.githubusercontent.com/8428372/59548283-f711b300-8f87-11e9-9588-556889f7a456.png)

**Comment by HemingwayLee** [comment]
![sa](https://user-images.githubusercontent.com/8428372/59549322-c2f1be80-8f96-11e9-94c3-21eb06cd785b.png)

**Comment by HemingwayLee** [comment]
![w2v](https://user-images.githubusercontent.com/8428372/61599911-4f636100-ac68-11e9-81fe-10110b37ba5f.png)

**Comment by HemingwayLee** [comment]
[KO_contract.pdf](https://github.com/HemingwayLee/test/files/3534217/KO_contract.pdf)

**Comment by HemingwayLee** [comment]
![lr](https://user-images.githubusercontent.com/8428372/63634451-95f61200-c691-11e9-950a-4620960a74e0.png)

**Comment by HemingwayLee** [comment]
![lr_lr](https://user-images.githubusercontent.com/8428372/63634679-d1dea680-c694-11e9-9692-603da58f89d0.png)

**Comment by HemingwayLee** [comment]
![softmax](https://user-images.githubusercontent.com/8428372/63634756-c8097300-c695-11e9-98da-b1319e802afc.png)

**Comment by HemingwayLee** [comment]
![countbased](https://user-images.githubusercontent.com/8428372/63647079-3f530b80-c757-11e9-8d8b-6043821589fc.png)

**Comment by HemingwayLee** [comment]
![countbased2](https://user-images.githubusercontent.com/8428372/63647310-fc466780-c759-11e9-9185-67bbcd9ffdd8.png)

**Comment by HemingwayLee** [comment]
![countbase_vs_w2v](https://user-images.githubusercontent.com/8428372/63647344-a0301300-c75a-11e9-8271-17615fb25196.png)

**Comment by HemingwayLee** [comment]
![comparison](https://user-images.githubusercontent.com/8428372/63648050-e8543300-c764-11e9-876f-2351939f7def.png)

**Comment by HemingwayLee** [comment]
![bean_search](https://user-images.githubusercontent.com/8428372/64489917-2b4cf500-d293-11e9-8712-e518db13e966.png)
![Out-of-Distribution-Examples-Detection](https://user-images.githubusercontent.com/8428372/64489918-2b4cf500-d293-11e9-9157-82f89b089f69.png)
![clm](https://user-images.githubusercontent.com/8428372/64489919-2be58b80-d293-11e9-86ab-0191cd0ed26c.png)
![lm](https://user-images.githubusercontent.com/8428372/64489921-2be58b80-d293-11e9-8213-e920708576b3.png)

**Comment by HemingwayLee** [comment]
![2018_ml_lib](https://user-images.githubusercontent.com/8428372/66694136-740d2900-eceb-11e9-8470-8ee0a608f64d.png)

**Comment by HemingwayLee** [comment]
![ULMfit](https://user-images.githubusercontent.com/8428372/66695266-7f1b8580-ecfa-11e9-8a83-d8fb93feb066.png)

**Comment by HemingwayLee** [comment]
<img width="574" alt="seq2seq_attention" src="https://user-images.githubusercontent.com/8428372/66698525-5b176e80-ed11-11e9-94af-0824d0799e8d.png">

**Comment by HemingwayLee** [comment]
![RNN_LSTM_GRU](https://user-images.githubusercontent.com/8428372/66712133-1fdd7400-edd3-11e9-9019-d9e434f58e24.png)

**Comment by HemingwayLee** [comment]
<img width="119" alt="building" src="https://user-images.githubusercontent.com/8428372/67139079-843e7e80-f27e-11e9-9807-175f28fc6480.png">

**Comment by HemingwayLee** [comment]
![table](https://user-images.githubusercontent.com/8428372/67216558-5dd73980-f45e-11e9-9352-90c38429f293.png)

**Comment by HemingwayLee** [comment]
![table](https://user-images.githubusercontent.com/8428372/68086331-186b3100-fe8e-11e9-88cd-842302b68fa0.png)

**Comment by HemingwayLee** [comment]
![val_test](https://user-images.githubusercontent.com/8428372/70388987-e7d26600-19fc-11ea-8f0e-4a5576cae22a.png)

**Comment by HemingwayLee** [comment]
![cross_validation](https://user-images.githubusercontent.com/8428372/70389197-78aa4100-19ff-11ea-880c-3fd029a320cd.png)

**Comment by HemingwayLee** [comment]
![hinge](https://user-images.githubusercontent.com/8428372/71305515-d768b500-2418-11ea-801c-37cc99cc02e5.png)

**Comment by HemingwayLee** [comment]
![hinge_loss_multiple](https://user-images.githubusercontent.com/8428372/71305643-662a0180-241a-11ea-8499-316cbcc159e7.png)

**Comment by HemingwayLee** [comment]
![all_loss](https://user-images.githubusercontent.com/8428372/71305997-7cd25780-241e-11ea-8d5b-96f8465473f9.png)

**Comment by HemingwayLee** [comment]
![Regularization](https://user-images.githubusercontent.com/8428372/71308626-650cca80-2442-11ea-99dc-2ae058619a47.png)

**Comment by HemingwayLee** [comment]
![overfit-ubderfit_orig](https://user-images.githubusercontent.com/8428372/71310079-35b28980-2453-11ea-85ce-1165c24fd9ae.png)

**Comment by HemingwayLee** [comment]
<img width="1008" alt="recap" src="https://user-images.githubusercontent.com/8428372/71315310-a928bb00-2494-11ea-9104-41e614f1948a.png">

**Comment by HemingwayLee** [comment]
![bias_variance](https://user-images.githubusercontent.com/8428372/71318902-73a6c080-24da-11ea-95d5-a6c68b07544f.png)

**Comment by HemingwayLee** [comment]
![cnn_visual](https://user-images.githubusercontent.com/8428372/71608388-505bcf80-2bc4-11ea-82fa-2da154ef9446.png)

**Comment by HemingwayLee** [comment]
![all_positive](https://user-images.githubusercontent.com/8428372/71760726-7e634b80-2f05-11ea-9279-f33d05c0b200.png)

**Comment by HemingwayLee** [comment]
![data_proc](https://user-images.githubusercontent.com/8428372/71760750-e7e35a00-2f05-11ea-9600-368e16a1fff5.png)

**Comment by HemingwayLee** [comment]
![gap](https://user-images.githubusercontent.com/8428372/71763372-9f846600-2f1e-11ea-8eab-4ac879f346f0.png)

**Comment by HemingwayLee** [comment]
![batch_normalization](https://user-images.githubusercontent.com/8428372/71763415-191c5400-2f1f-11ea-9630-77dfa250fe9f.png)

**Comment by HemingwayLee** [comment]
![sgd_problem](https://user-images.githubusercontent.com/8428372/72125333-93c7f200-33ab-11ea-99a9-50d11721e048.png)

**Comment by HemingwayLee** [comment]
![sgd_problem_2](https://user-images.githubusercontent.com/8428372/72125381-bb1ebf00-33ab-11ea-9492-c7afae673697.png)

**Comment by HemingwayLee** [comment]
![momentum](https://user-images.githubusercontent.com/8428372/72128020-d2ae7580-33b4-11ea-802b-7fa51099a9b1.png)

**Comment by HemingwayLee** [comment]
![momentum2](https://user-images.githubusercontent.com/8428372/72128462-62a0ef00-33b6-11ea-9180-ce159031b02b.png)

**Comment by HemingwayLee** [comment]
![computational_graph](https://user-images.githubusercontent.com/8428372/72130257-4bfd9680-33bc-11ea-975b-a39dc219a58d.png)

**Comment by HemingwayLee** [comment]
![leaky_relu](https://user-images.githubusercontent.com/8428372/72134858-650c4480-33c8-11ea-8d25-c166a41576dd.png)

**Comment by HemingwayLee** [comment]
![l1_l2_term](https://user-images.githubusercontent.com/8428372/72135633-50c94700-33ca-11ea-9655-dda1d8ba7946.png)

**Comment by HemingwayLee** [comment]
![l1_l2_visualization](https://user-images.githubusercontent.com/8428372/72136127-6f7c0d80-33cb-11ea-8879-e68515da53ee.png)

**Comment by HemingwayLee** [comment]
![boxing](https://user-images.githubusercontent.com/8428372/72200071-1aea9800-3488-11ea-9431-ead2006cc459.png)
![breakout](https://user-images.githubusercontent.com/8428372/72200072-1cb45b80-3488-11ea-8554-b21803f41c93.png)

**Comment by HemingwayLee** [comment]
![ex2](https://user-images.githubusercontent.com/8428372/74094335-e28bb780-4b22-11ea-8448-aa2c7dadfe95.png)
![ex1](https://user-images.githubusercontent.com/8428372/74094337-e4557b00-4b22-11ea-9fc8-dd2ef509a6fd.png)

**Comment by HemingwayLee** [comment]
![Markov2](https://user-images.githubusercontent.com/8428372/74341678-7394b400-4deb-11ea-8be5-a3662b5f37f5.png)
![Markov1](https://user-images.githubusercontent.com/8428372/74341692-78596800-4deb-11ea-8e4c-f9e47f01f8e9.png)

**Comment by HemingwayLee** [comment]
![return](https://user-images.githubusercontent.com/8428372/74606766-6b53b600-5116-11ea-831f-eabd51156a88.png)

**Comment by HemingwayLee** [comment]
![value_fun](https://user-images.githubusercontent.com/8428372/74616561-e4312d00-516b-11ea-9809-a543a4ba78ce.png)

**Comment by HemingwayLee** [comment]
![MRP_value_func](https://user-images.githubusercontent.com/8428372/74616663-923cd700-516c-11ea-89de-27357ebe7af2.png)

**Comment by HemingwayLee** [comment]
![MDP](https://user-images.githubusercontent.com/8428372/74616811-50606080-516d-11ea-8441-c1643fbbbf8e.png)

**Comment by HemingwayLee** [comment]
![p_in_mdp_with_2_a](https://user-images.githubusercontent.com/8428372/74616895-be0c8c80-516d-11ea-8087-1f9280894aae.png)

**Comment by HemingwayLee** [comment]
![qlearning_process](https://user-images.githubusercontent.com/8428372/76696665-3022b380-66d1-11ea-8bf1-a8c92c761e0f.png)
![update_qvalue](https://user-images.githubusercontent.com/8428372/76696667-31ec7700-66d1-11ea-8d2a-6ec1cea8a8d7.png)

**Comment by HemingwayLee** [comment]
![python_hash](https://user-images.githubusercontent.com/8428372/84723476-a381dd00-afc0-11ea-8131-d8a94b9d957b.png)

**Comment by HemingwayLee** [comment]
![csrf](https://user-images.githubusercontent.com/8428372/84728084-6b33cc00-afcb-11ea-8954-b52a10213136.png)


https://user-images.githubusercontent.com/8428372/84728084-6b33cc00-afcb-11ea-8954-b52a10213136.png

**Comment by HemingwayLee** [comment]
![pass](https://user-images.githubusercontent.com/8428372/90464008-51d40b00-e147-11ea-86a0-a5f5a175e050.png)

**Comment by HemingwayLee** [comment]
![hashtable](https://user-images.githubusercontent.com/8428372/102005952-3c22cb00-3d60-11eb-9924-145c7e2f872f.png)
![partition](https://user-images.githubusercontent.com/8428372/102005954-3d53f800-3d60-11eb-9444-d779d610f973.png)
![replication](https://user-images.githubusercontent.com/8428372/102005955-3e852500-3d60-11eb-9ef6-82a3e2690135.png)

**Comment by HemingwayLee** [comment]
![1609223332806](https://user-images.githubusercontent.com/8428372/104396534-305c3b80-558e-11eb-8a19-7fcc6efa92a3.JPEG)

**Comment by HemingwayLee** [comment]
![k8s-vs-compose-fig-02](https://user-images.githubusercontent.com/8428372/169322926-e22f03d8-18a0-4824-b7aa-72ec98859226.png)
![k8s-vs-compose-fig-01](https://user-images.githubusercontent.com/8428372/169322936-3c9e28b5-9f6e-432f-ab97-4b7a2e792381.png)
